tools/notification-configuration/common/Services/AzureDevOpsService.cs

```diff
@@ -122,7 +123,35 @@ public async Task<IEnumerable<WebApiTeam>> GetTeamsAsync(string projectName, int
 
```

Comment:
It is probably worth making this method internal so people don't accidently start using it and not thinking about the paging.